src/com/strongjoshua/console/AbstractConsole.java

```diff
@@ -115,6 +115,8 @@ public void setCommandExecutor (CommandExecutor commandExec) {
 	 */
 	@Override
 	public void execCommand(String command) {
+		if(disabled) return;
```

Comment:
This enforces the disabled flag, whether it is a GUIConsole, HeadlessConsole, or even a custom Console extended from AbstractConsole, it is disabled from executing commands.